internal/providers/terraform/ibm/is_instance.go

```diff
@@ -25,14 +25,14 @@ func newIsInstance(d *schema.ResourceData, u *schema.UsageData) *schema.Resource
 	isDedicated := !((dedicatedHost == "") && (dedicatedHostGroup == ""))
 	name := d.Get("name").String()
 
-	boot_volume := make([]struct {
+	bootVolume := make([]struct {
 		Name string
 		Size int64
 	}, 0)
 
-	boot_volume_parse := d.Get("boot_volume").Array()
-	if len(boot_volume_parse) > 0 {
-		for _, volume := range boot_volume_parse {
+	bootVolumeParse := d.Get("boot_volume").Array()
+	if len(bootVolumeParse) > 0 {
```

Comment:
if a boot volume isn't sepcified, then a default one will be created and will need to be added as a cost component. see https://cloud.ibm.com/docs/vpc?topic=vpc-creating-virtual-servers&interface=ui 
<img width="767" alt="Screenshot 2024-09-25 at 1 59 03 PM" src="https://github.com/user-attachments/assets/e278e3e7-6b40-45e3-9c68-29ce8e314392">